src/openvic-simulation/utility/Getters.hpp

```diff
@@ -196,12 +196,12 @@ namespace OpenVic {
  *		CultureGroup const& PROPERTY(group);// CultureGroup const& group;	CultureGroup const& get_group() const;
  *		Province& PROPERTY(province);		// Province& province;			Province const& get_province() const;
  */
-#define PROPERTY(NAME) PROPERTY_ACCESS(NAME, private)
-#define PROPERTY_CUSTOM_PREFIX(NAME, PREFIX) PROPERTY_CUSTOM_NAME(NAME, PREFIX##_##NAME)
-#define PROPERTY_CUSTOM_NAME(NAME, GETTER_NAME) PROPERTY_FULL(NAME, GETTER_NAME, private)
-#define PROPERTY_ACCESS(NAME, ACCESS) PROPERTY_FULL(NAME, get_##NAME, ACCESS)
-#define PROPERTY_FULL(NAME, GETTER_NAME, ACCESS) \
-	NAME; \
+#define PROPERTY(NAME, ...) PROPERTY_ACCESS(NAME, private, __VA_ARGS__)
+#define PROPERTY_CUSTOM_PREFIX(NAME, PREFIX, ...) PROPERTY_CUSTOM_NAME(NAME, PREFIX##_##NAME, __VA_ARGS__)
+#define PROPERTY_CUSTOM_NAME(NAME, GETTER_NAME, ...) PROPERTY_FULL(NAME, GETTER_NAME, private, __VA_ARGS__)
+#define PROPERTY_ACCESS(NAME, ACCESS, ...) PROPERTY_FULL(NAME, get_##NAME, ACCESS, __VA_ARGS__)
+#define PROPERTY_FULL(NAME, GETTER_NAME, ACCESS, ...) \
+	NAME __VA_OPT__(=) __VA_ARGS__; \
```

Comment:
```suggestion
	NAME __VA_OPT__({ __VA_ARGS__ }); \
```
Are we sure we always want to use `=` here, rather than curly brackets or even just regular round brackets? I guess you could always do `PROPERTY(name, { 1, 2, 3 });`, but are we sure `var name = { ... };` is always equivalent to `var name { ... };`?